examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -118,6 +120,31 @@ def _blew_up(results: List[SimulationResult]) -> bool:
     return np.isnan(du_dls).any() or (du_dls.std(1).max() > 1000)
 
 
+class RBFETransformIndex:
+    """Builds an index of relative free energy transformations to use
+    with construct_mle_layer
+    """
+
+    def __len__(self):
+        return len(self._indices)
+
+    def __init__(self):
+        self._indices = {}
+
+    def build(self, refs: List[RelativeFreeEnergy]):
+        for ref in refs:
+            self.get_transform_indices(ref)
+
+    def get_transform_indices(self, ref: RelativeFreeEnergy) -> List[int]:
```

Comment:
nit: should the type be
```suggestion
    def get_transform_indices(self, ref: RelativeFreeEnergy) -> Tuple[int, int]:
```